### PR TITLE
Closes #1533 xASL_im_CreateGroupAnalysisMask: improve missing templates warnings

### LINKED ISSUE #1533: xASL_im_CreateGroupAnalysisMask improve missing templates warnings
### Description

Minor improvements of xASL_im_CreateGroupAnalysisMask

### Tasks

- [x] xASL_im_CreateGroupAnalysisMask: improve missing templates warnings

### Release notes

Report with #870

### PR DESCRIPTION
**Pull request by HenkMutsaerts**
### Linked issue

Closes #1533

### CODE REVIEW

File: Modules/Module_Processing/Module_Population/xASL_im_CreateGroupAnalysisMask.m [hunk 1]
```diff
@@ -44,58 +44,53 @@
...
 PathT1 = xASL_adm_GetFileList(x.D.TemplatesStudyDir, ['^T1' x.S.TemplateNumberName '_bs-mean\.nii$'], 'FPList', [1 1]);
 
-if ~isempty(PathTemplateSusceptibilityMask); PathTemplateSusceptibilityMask = PathTemplateSusceptibilityMask{1}; end
-if ~isempty(PathFoV); PathFoV = PathFoV{1}; end
-if ~isempty(PathVascularMask); PathVascularMask = PathVascularMask{1}; end
-if ~isempty(PathpGM); PathpGM = PathpGM{1}; end
-if ~isempty(PathpWM); PathpWM = PathpWM{1}; end
-if ~isempty(PathpCSF); PathpCSF = PathpCSF{1}; end
-if ~isempty(PathT1); PathT1 = PathT1{1}; end
+pathsTemplates = {'PathTemplateSusceptibilityMask' 'PathFoV' 'PathVascularMask' 'PathpGM' 'PathpWM' 'PathpCSF' 'PathT1'};
```
Comment (jan-petr) [inline Modules/Module_Processing/Module_Population/xASL_im_CreateGroupAnalysisMask.m:67]: You have just deleted PathpGM, PathpWM, and PathpCSF from the code...

**Review by jan-petr — CHANGES_REQUESTED** [review]
One comment.

File: Modules/Module_Processing/Module_Population/xASL_im_CreateGroupAnalysisMask.m (hunk 1 above)
Comment (HenkMutsaerts) [reply Modules/Module_Processing/Module_Population/xASL_im_CreateGroupAnalysisMask.m:67]: Yes that was accidentally indeed, now put back

**Review by jan-petr — APPROVED** [review]
OK.

**Pull request merged**